quic/s2n-quic-events/src/main.rs

```diff
@@ -136,10 +136,55 @@ impl EventInfo<'_> {
             feature_alloc: quote!(),
         }
     }
+
+    fn test_tls_events() -> Self {
```

Comment:
I would probably include a comment about what these events are for. It's a little different than s2n-quic/dc-quic because they will only be used for testing.